src/main/java/dao/impl/BasketDaoHibImpl.java

```diff
@@ -120,4 +120,4 @@ public void clearBasket(User user) {
         }
         return Optional.empty();
     }
-}
\ No newline at end of file
+}
```

Comment:
try to use hql here?

Reply:
hql query not working there, because I setting user.id, which not exist in the products table

Reply:
why we use product id and description in basket model? we should use List of products and related table (many-to-many)

Reply:
It's option for user  basket. It's feature. ))